### ISSUE TITLE
Tooltips could be shown on hover when app descriptions are cut by an ellipsis

### ISSUE THREAD

**Issue by somas95** [flathub/linux-store-frontend#113]
On the pages were multiple applications are shown, it would be nice to have an on hover tooltip to read the full description without having to enter to the app page

![image](https://user-images.githubusercontent.com/8913975/40456634-b7c5d378-5ef2-11e8-907b-a4fb3b40cabf.png)

**Comment by allanday** [comment]
I agree that the cut off descriptions don't look very good, but I don't think that tooltips are a good solution: you would still see the cut off descriptions, so it still wouldn't look very good.

The obvious solution would be to leave enough space for two lines of description text. Alternatively, we might want to consider dropping the descriptions altogether.

**Comment by jgarciao** [comment]
At the begining we tried tooltips in the descriptions but we disabled them because they were very annoying. 

We are planning to revisit the tiles design. Maybe we'll do them wider and/or having two lines. The alternative solutions seems fine too

**Comment by anovsiradj** [comment]
+1 yes, but I suggest using just "title" attribute, will solving the problem.

*(sory for my english)*
